### ISSUE TITLE
Records absolute path instead of path from content root

### ISSUE THREAD

**Issue by devurandom** [wakatime/jetbrains-wakatime#301]
When working on a Clojure project (deps.edn, with Cursive plugin), WakaTime shows me absolute paths on https://wakatime.com/projects/[REDACTED] in the tooltips, instead of what IntelliJ seems to call "Path From Content Root" or "Path From Repository Root" (right click on a file in the "Project" side bar, "Copy Path/Reference...").  Thus the records will be different for different people working on the same project, when moving the files around new records will be started, and the filenames in the tooltips are long and inconvenient.

**Comment by gandarez** [comment]
@devurandom you are right, the balloons shows the filename sorted by time spent whereas the tooltip shows the full filepath of each balloon. Can you attach an screenschot of the the problem you mentioned when different people start working on the same project?

**Comment by alanhamlett** [comment]
That's why heartbeats have a `project_root_count` attribute, which is the count of slashes needed to reach the project's root path. We use that when comparing absolute paths between team members.

**Comment by devurandom** [comment]
> That's why [heartbeats](https://wakatime.com/developers/#heartbeats) have a `project_root_count` attribute, which is the count of slashes needed to reach the project's root path.

I feel uncomfortable with local directory structures being saved.  I would prefer to have more control over what exactly is saved -- and this being a bit more predictable.

> We use that when comparing absolute paths between team members.

Thanks, that relieves my concern when testing how this would work for a whole team.
